src/styles/font-override-classes/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Font override classes
+description: Use paragraphs for your core content
+section: Styles
+theme: Typography
+layout: layout-pane.njk
+showPageNav: true
+order: 11
+---
+
+{% from "_example.njk" import example %}
+
+Use font override classes sparingly to etc. etc.
```

Comment:
This obvs needs writing up properly... is this enough?

```suggestion
Use font override classes sparingly to change the default styling of text on a page.
```

Reply:
Is it too strong to say something like 'Only use font override classes to change the default styling of text on a page' ? Just thinking that 'sparingly' might be too vague.

Reply:
I think that probs wouldn't be quite accurate... Font override classes change the default styling – that's what they do, so the 'only' wouldn't make sense. I guess the 'sparingly' is getting at the idea that it's better to use the default styles first, and only override them if they're not working for your use case. The more we override stuff, the more inconsistency there is across GOV.UK.

Reply:
Fair enough.

Reply:
Decision: this is fine 🤘